zenodo/modules/support/templates/zenodo_support/contact_form.html

```diff
@@ -106,6 +106,31 @@ <h3 class="upload-header">Drag files anywhere or click <a class="upload-button">
           </form>
         </div>
       </div>
+
+      <!-- Modal -->
+      <div class="modal fade" id="confirmationModal" tabindex="-1" role="dialog" aria-labelledby="confirmationModalLabel">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+              <h4 class="modal-title" id="confirmationModalLabel">Warning!</h4>
+            </div>
+            <div class="modal-body">
+              <form>
+                <div class="form-group">
+                  <p>You haven't attached any files. If you want to add/update files, make sure you attach the files
```

Comment:
If you want to add/update files, make sure to attach them or provide us with the link to them.